tools/msvs/find_nasm.cmd

```diff
@@ -1,24 +1,24 @@
 @IF NOT DEFINED DEBUG_HELPER @ECHO OFF
 
-ECHO Looking for NASM
+echo Looking for NASM
 
-FOR /F "delims=" %%a IN ('where nasm 2^> NUL') DO (
-  EXIT /B 0
+for /f "delims=" %%a IN ('where nasm 2^> NUL') DO (
+  echo NASM found in %%a
+  exit /b 0
 )
 
-IF EXIST "%ProgramFiles%\NASM\nasm.exe" (
-  SET "Path=%Path%;%ProgramFiles%\NASM"
-  EXIT /B 0
+for %%a in ("%ProgramFiles%\NASM\nasm.exe" "%ProgramFiles(x86)%\NASM\nasm.exe" "%LOCALAPPDATA%\bin\NASM\nasm.exe") do (
+  if exist %%a (
+    call :find-nasm %%a
+    exit /b 0
+  )
 )
+exit /b 1
 
-IF EXIST "%ProgramFiles(x86)%\NASM\nasm.exe" (
-  SET "Path=%Path%;%ProgramFiles(x86)%\NASM"
-  EXIT /B 0
-)
-
-if EXIST "%LOCALAPPDATA%\bin\NASM\nasm.exe" (
-  SET "Path=%Path%;%LOCALAPPDATA%\bin\NASM"
-  EXIT /B 0
-)
-
-EXIT /B 1
+:find-nasm
+set p=%~1
+set p=%p:~0,-9%
```

Comment:
remove the last nine characters, for example, `C://xxx/nasm.exe` will become `C://xxx`

Reply:
You can add that comment to the script